src/main/java/com/google/sps/app/servlets/LogIn.java

```diff
@@ -0,0 +1,62 @@
+package com.google.sps.app.servlets;
+
+import java.io.IOException;
+import java.util.UUID;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.cloud.datastore.BaseEntity;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Key;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.sps.util.Validator;
+
+@WebServlet("/LogIn")
+public class LogIn extends HttpServlet {
+
+    final String textInput = "text-input-";
```

Comment:
Sorry about not being clear. I meant

private static final String USER_NAME_PARAM = 
" text-input-user-name"

Reply:
OH. I understand now! I can see why that is an important addition.